demos/atk-init.php

```diff
@@ -36,7 +36,7 @@
 $app->title = 'Agile UI Demo v' . $app->version;
 
 
-if (file_exists(dirname(__DIR__) . '/public/atkjs-ui.min.js')) {
+if (file_exists(__DIR__ . '/../public/atkjs-ui.min.js')) {
```

Comment:
Dirname here was ok, but it does not matter here and this is fine. I try to use `dirname` when the output is returned as the path looks nicer/more normalized.

Reply:
Ok - good to know.  I am always mixt up when to use what. 😄 Should we keep it this way? I do not mind to change back.

Reply:
I think with `. /../` it is more readable, so the current form is preffered by me

Reply:
revert